src/stories/HdForm.stories.js

```diff
@@ -72,6 +74,10 @@ const Template = (args, { argTypes }) => ({
       }));
       console.log('submitted', params);
     },
+    onFormChange(val) {
+      this.hasFormChanged = val;
+      console.log('hasChanged', val);
```

Comment:
Don't forget to remove the `console.logs` 😄

Reply:
This was actually suggested by Vini, so we can see logs under actions when trying out the component https://github.com/homeday-de/homeday-blocks/pull/647#discussion_r564651065 :)